Effects/WeaponEffect.cs

```diff
@@ -0,0 +1,331 @@
+﻿using Loot.Modifiers;
+using Microsoft.Xna.Framework;
+using System;
+using Terraria;
+using Terraria.ModLoader.IO;
+
+namespace Loot.Effects.WeaponEffects
+{
+	public class DamagePlusEffect : ModifierEffect
```

Comment:
Though I understand ease of access for effects to be in a single file, it is favorable for them to be in separate classes. This keeps the size per file fairly small, and excludes code from other effects that may not be of interest.